libs/genai/langchain_google_genai/_function_utils.py

```diff
@@ -160,17 +149,17 @@ def _convert_tool_to_genai_function(tool: BaseTool) -> FunctionDeclaration:
 
 
 def _convert_pydantic_to_genai_function(
-    pydantic_model: Type[BaseModel],
+    pydantic_model: Type[BaseModel], tool_name: str = ""
```

Comment:
nits: can we declare tool_name as `Optional[str]=None`

Reply:
fixed it.